Buscorepuestos/buscorepuestos-web
# Feature/br 355 card price

## What type of PR is this? (check all applicable)


## Description
The design adapts to the Figma, and the title where each swiper goes is also redesigned and adapted.


## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-355/card-price

## QA Instructions, Screenshots, Recordings

verify card price component

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
